src/plugins/intel_cpu/tests/unit/streams_info/streams_info_table_test.cpp

```diff
@@ -1818,11 +1970,172 @@ StreamsCalculationTestCase _1sockets_mock_tput_3 = {
+    {{60, 60, 0, 0, -1, -1}, {40, 40, 0, 0, 0, 0}, {20, 20, 0, 0, 1, 1}},
+    {{1, MAIN_CORE_PROC, 10, 1, 1}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_5 = {
+    1,
+    false,
+    50,
+    0,
+    0,
+    1,
+    "LATENCY",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 60, 0, 0, -1, -1}, {40, 40, 0, 0, 0, 0}, {20, 20, 0, 0, 1, 1}},
+    {{1, ALL_PROC, 50, -1, -1}, {0, MAIN_CORE_PROC, 20, 1, 1}, {0, MAIN_CORE_PROC, 30, 0, 0}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_6 = {
+    1,
+    false,
+    0,
+    0,
+    0,
+    1,
+    "LATENCY",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 0, 60, 0, -1, -1}, {40, 0, 40, 0, 0, 0}, {20, 0, 20, 0, 1, 1}},
+    {{1, EFFICIENT_CORE_PROC, 20, 1, 1}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_7 = {
+    1,
+    true,
+    0,
+    0,
+    0,
+    1,
+    "THROUGHPUT",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 0, 60, 0, -1, -1}, {40, 0, 40, 0, 0, 0}, {20, 0, 20, 0, 1, 1}},
+    {{1, EFFICIENT_CORE_PROC, 20, 1, 1}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_8 = {
+    1,
+    false,
+    10,
+    0,
+    0,
+    1,
+    "LATENCY",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 0, 60, 0, -1, -1}, {40, 0, 40, 0, 0, 0}, {20, 0, 20, 0, 1, 1}},
+    {{1, EFFICIENT_CORE_PROC, 10, 1, 1}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_9 = {
+    1,
+    false,
+    50,
+    0,
+    0,
+    1,
+    "LATENCY",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 0, 60, 0, -1, -1}, {40, 0, 40, 0, 0, 0}, {20, 0, 20, 0, 1, 1}},
+    {{1, ALL_PROC, 50, -1, -1}, {0, EFFICIENT_CORE_PROC, 20, 1, 1}, {0, EFFICIENT_CORE_PROC, 30, 0, 0}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_10 = {
+    1,
+    false,
+    0,
+    0,
+    0,
+    1,
+    "LATENCY",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 30, 0, 30, -1, -1}, {40, 20, 0, 20, 0, 0}, {20, 10, 0, 10, 1, 1}},
+    {{1, ALL_PROC, 20, 1, 1}, {0, MAIN_CORE_PROC, 10, 1, 1}, {0, HYPER_THREADING_PROC, 10, 1, 1}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_11 = {
+    1,
+    true,
+    0,
+    0,
+    0,
+    1,
+    "THROUGHPUT",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 30, 0, 30, -1, -1}, {40, 20, 0, 20, 0, 0}, {20, 10, 0, 10, 1, 1}},
+    {{1, ALL_PROC, 20, 1, 1}, {0, MAIN_CORE_PROC, 10, 1, 1}, {0, HYPER_THREADING_PROC, 10, 1, 1}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_12 = {
+    1,
+    false,
+    15,
+    0,
+    0,
+    1,
+    "LATENCY",
+    ov::intel_cpu::Config::LatencyThreadingMode::PER_SOCKET,
+    {{60, 30, 0, 30, -1, -1}, {40, 20, 0, 20, 0, 0}, {20, 10, 0, 10, 1, 1}},
+    {{1, ALL_PROC, 15, 1, 1}, {0, MAIN_CORE_PROC, 10, 1, 1}, {0, HYPER_THREADING_PROC, 5, 1, 1}},
+};
+StreamsCalculationTestCase _2sockets_mock_latency_13 = {
```

Comment:
How about 4 or 8 socket systems?

Reply:
Added more mock test cases for 4 and 8 socket systems